bundles/AdminBundle/Security/Authenticator/AdminAbstractAuthenticator.php

```diff
@@ -131,7 +132,7 @@ public function onAuthenticationSuccess(Request $request, TokenInterface $token,
     }
 
     /**
-     * @param $user
+     * @param User|null $user
```

Comment:
```suggestion
     * @param User $user
```
passing null has no effect so I would avoid that.